tuf/api/metadata.py

```diff
@@ -431,16 +431,17 @@ def __init__(
         expires: datetime,
         unrecognized_fields: Optional[Mapping[str, Any]] = None,
     ):
+        # Accept semver (X.Y.Z) but also X.Y for legacy compatibility
         spec_list = spec_version.split(".")
-        if (
-            len(spec_list) != 3
-            or not all(el.isdigit() for el in spec_list)
-            or spec_list[0] != SPECIFICATION_VERSION[0]
+        if len(spec_list) not in [2, 3] or not all(
+            el.isdigit() for el in spec_list
```

Comment:
this is a weird construct but black wants to lay it out this way -- I'll take suggestions to make it clearer though

Reply:
I don't have a better suggestion as well, but it's not so bad.
black splits `all` brackets so that we can see clearly what are the arguments which are the complicated part.